### ISSUE TITLE
Contact Center Review for [Authenticated Experience - Direct Deposit for EDU]

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#19899]
# Request the Contact Center Review 

## Description
*Please provide the name(s) of your product and a high level description of your product.*
[Direct Deposit for EDU benefits product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/direct-deposit/edu-direct-deposit/README.md)

### Launch Date/Release plan
[Direct deposit for EDU release plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/direct-deposit/edu-direct-deposit/product/Release%20Plan.md)

### Version Number
*VA.gov Profile Product Guide V2*

### Product Guide
*[VA.gov Profile Product Guide V2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Call%20Center/VA%20Profile%20-%20Product%20Guide%20v2.docx)*

### Product Video
*N/A*

### Product Video Transcript
*N/A*

## Acceptance Criteria
- [ ] VSP Contact Center QA has been completed 
- [ ] VSP Contact Center sends artifacts to appropriate contact centers
- [ ] VSP: [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1d219oL1zCvCvnv1Bx-dI-GMzwgbarLv9_bzMSa3ULjA/edit#gid=1710283887) is updated

**Comment by Kimberley2019** [comment]
Contact Center is in receipt and will review.
I have informed @mshea0606  I do not think that a product video is required for this; however, once our team reviews, if one is needed we will advise.
@starlow00 please be sure that VEO requests are included.

**Comment by Kimberley2019** [comment]
@mshea0606, @starlow00  should be finished reviewing the guide late today or early tomorrow.  It will then be sent to the VEO call center for their review by COB tomorrow.  We will then give them 3 to 5 business days to review and provide any feedback.

The ticket will be updated with the modified guide before sending to VEO.

**Comment by starlow00** [comment]
Hi @mshea0606 (cc @Kimberley2019) Attached is the draft of the product guide with my edits, which were minor. I think it works well!
[VA Profile - Product Guide v3_ST_021821_.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/6006157/VA.Profile.-.Product.Guide.v3_ST_021821_.docx)

**Comment by starlow00** [comment]
Sent to VEO for review, requesting reply by COB 2/25/21.

**Comment by mshea0606** [comment]
Hey @starlow00, do you have any update regarding the review?

**Comment by starlow00** [comment]
VEO deadline has passed without additional edits. Closing after attaching latest copy here.

**Comment by starlow00** [comment]
@mshea0606 Latest clean copy that's VEO approved is attached. Closing ticket.
[VA Profile - Product Guide v3_030121.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/6063926/VA.Profile.-.Product.Guide.v3_030121.docx)

**Comment by Samara-Strauss** [comment]
Hi all — Yesterday was Matt's last day on the team, so I am stepping in until we have a new PM to make sure things are moving along.

I am reopening this issue temporarily to check in on the status of everything. Is there anything else the contact center team needs from us? Given that this issue was closed, I am assuming that the Profile Guide has been fully reviewed and approved, and that it has also been handed off to contact centers. Is that accurate? Is there anything else this team needs with regards to the contact center guide updates for the direct deposit for education functionality we've added to the profile?

If I can get confirmation on this, I'll close this issue again and consider our engagement for this project complete!
